test/selenium/configurations.rb

```diff
@@ -36,11 +36,11 @@ def common_configuration_for_firefox_extension
   Capybara.app_host = "chrome://privly"
   @@privly_applications_folder_path = Capybara.app_host + "/content/privly-applications/"
   puts "Packaging the Firefox Extension"
-  system( "cd ../../../../../ && pwd && ./package.sh && cd chrome/content/privly-applications/test/selenium" )
+  system("cd ../../../../../ && pwd && jpm xpi && cd chrome/content/privly-applications/test/selenium")
```

Comment:
When we submit the extension to Mozilla for distribution, we remove many files that are needed for development but not for the user. `jpm xpi` probably includes everything, so it doesn't mirror our deployment process. Is there a way to make jpm exclude certain files?